hazelcast/src/main/java/com/hazelcast/map/impl/querycache/subscriber/SubscriberAccumulatorHandler.java

```diff
@@ -57,13 +66,28 @@ public void handle(QueryCacheEventData eventData, boolean ignored) {
             case EVICTED:
                 queryCache.deleteInternal(keyData, false, entryEventType);
                 break;
-            // TODO: if we want strongly consistent clear & evict, removal can be made based on sequence and partition ID
             case CLEAR_ALL:
+                handleMapWideEvent(eventData, entryEventType, evictAllCount, evictedEntryCount);
```

Comment:
Seems like this line should be swapped with the line in the next `case`.

Reply:
good catch, fixed